cloudpathlib/s3/s3client.py

```diff
@@ -258,6 +262,8 @@ def _upload_file(self, local_path: Union[str, os.PathLike], cloud_path: S3Path)
         obj = self.s3.Object(cloud_path.bucket, cloud_path.key)
 
         extra_args = {}
```

Comment:
You can simplify this even further: `extra_args = self.boto3_upload_extra_args.copy()` (`__init__` ensures that it is a dict).

Reply:
Fixed

Reply:
Addressed in [4f29d1f](https://github.com/TerrawattEnergy/cloudpathlib/pull/1/commits/4f29d1f3941ccb08fcbeed95eef2d36c1be37985)